.github/workflows/package-pre-release.yml

```diff
@@ -94,7 +94,7 @@ jobs:
           path: dist
       - uses: actions/setup-node@v2
         with:
-          node-version: 15
+          node-version: 16
```

Comment:
Cool thing that we're moving up to node16. I think I remember that older versions of node/npm would create an old-style package-lock v1.

Reply:
Yes I think this was between [npm 6 and 7](https://github.blog/2021-02-02-npm-7-is-now-generally-available/#changes-to-the-lockfile). 
We only have one Issue to fix for the publish step #129 where we use npm 6. But we don't install anything here, it's just for the publish command so I hope this is still working with the newer node version.